pike/__init__.py

```diff
@@ -13,5 +13,3 @@
         'test',
         'transport',
 ]
```

Comment:
I might be getting too tedious, but is it worth commenting

    # version is in setup.py.setup["use_scm_version"]